Appraisals

```diff
@@ -38,6 +38,22 @@ if Gem::Version.new(RUBY_VERSION) > Gem::Version.new('2.5.0')
   appraise 'sidekiq61' do
     gem 'sidekiq', '~> 6.1.0'
   end
+
+  appraise 'sidekiq62' do
```

Comment:
You should run `bundle exec appraisal generate` and commit the `gemfiles`

Overall appraisal does:

```
BUNDLE_GEMFILE=gemfiles/sidekiq7.gemfile bundle install

BUNDLE_GEMFILE=gemfiles/sidekiq7.gemfile bundle exec rspec
```